static/app/routes.tsx

```diff
@@ -2064,7 +2051,7 @@ function buildRoutes() {
         )}
         key="org-issues-group-id"
       >
-        {issueDetailsChildRoutes({forCustomerDomain: false})}
+        {issueDetailsChildRoutes}
       </Route>
       {USING_CUSTOMER_DOMAIN && (
         <Route
```

Comment:
Afaict this route already has the `withDomaiNRequired` that is needed to get the redirect to happen no?)